index.php

```diff
@@ -24,13 +23,12 @@
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-    <header class="header"> 
-        <p class="header-title">Favorite Movies</p>
+    <header> 
+        <h1>Favorite Movies</h1>
     </header>
-        <section class="card-section">   
-            <?php foreach($allMoviesAsObjects as $movie) {
-                   echo $movie->createCard();
-                   } ?>
+    <section class="card-section">   
+         <?php foreach($allMoviesAsObjects as $movie)
+          {echo $movie->createCard();} ?>
```

Comment:
{ } around the echo is not PSR12 I think?

should be:
<?php 
foreach($allMoviesAsObjects as $movie) {
          echo $movie->createCard();
} 
?>